TESTS/filesystem/files/main.cpp

```diff
@@ -300,11 +298,11 @@ utest::v1::status_t test_setup(const size_t number_of_cases) {
 Case cases[] = {
     Case("File tests", test_file_tests),
     Case("Simple file test", test_simple_file_test),
-    Case("Small file test", test_write_file_test<32, 31, 29, "smallavacado">),
-    Case("Medium file test", test_write_file_test<8192, 31, 29, "mediumavacado">),
-    Case("Large file test", test_write_file_test<262144, 31, 29, "largeavacado">),
-    Case("Block Size file test", test_write_file_test<9000, 512, 512, "blockfile">),
-    Case("Multiple block size file test", test_write_file_test<26215, MBED_TEST_BUFFER, MBED_TEST_BUFFER, "bigblockfile">),
```

Comment:
Should these be "**Non**-block size file test"?

Reply:
Multiple block size file test - It is actually both, file size is not in multiple of blocks, but read/write are in multiple of blocks (`MBED_TEST_BUFFER`)

Reply:
Oh, what about the "Block size file test"? how is that different from "Multiple block size file test"? Is it just smaller?